### ISSUE TITLE
1.19.2 server placeholder issues (Paper-Spigot)

### ISSUE THREAD

**Issue by mike3132** [PlaceholderAPI/PlaceholderAPI#887]
### Confirmation

- [X] My issue isn't already found on the Issue tracker.
- [X] My issue is about **PlaceholderAPI** and not any expansion or external plugin
- [X] The issue isn't already fixed in a Spigot Release or Development Build.
- [X] The [Common Issues](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Common-Issues) page doesn't mention this issue.

### Type

Plugin Bug

### What happens?

Placeholder API (2.11.2)  using (Paper(138) 1.19.2-R0.1-SNAPSHOT Dosn't parse placeholders for downloaded expansions (ecloud) inside of holograms and is adding in extra % inside of CMI's Chat function. CMI version 9.2.3.7 CMILib version 1.2.3.6, Placeholders are also not parsing inside of other plugins including TAB (https://github.com/NEZNAMY/TAB).

Placholders in question that are not working:
Deluxe tags (https://www.spigotmc.org/resources/deluxetags.4390/)
SuperbVote (https://www.spigotmc.org/resources/superbvote.11626/)
LuckPerms (https://luckperms.net)
Voteparty (https://www.spigotmc.org/resources/voteparty-new-party-modes-all-in-one-voting-plugin-30-sale.987/)

There is no console errors upon start or /papi reload.

### Expected Behaviour

Parse placeholders.

### How to Reproduce

1. Install placeholder API 2.11.2 on a paper spigot 1.19.2 server.
2. Use /papi ecloud download to grab placeholders for the listed plugins above.
3. Use /papi reload
4. Add in the placeholders to cmi config.yml (For chat) and TAB
5. Restart server for changes to take effect.

### `/papi dump` Output

https://paste.helpch.at/uhipupesog

### Console Log

https://paste.helpch.at/anifikuboy.makefile

### Errors

N/A

### Additional Info

N/A

**Comment by mike3132** [comment]
@LightNetwork

**Comment by mike3132** [comment]
![wQNEwSSh](https://user-images.githubusercontent.com/32972166/193721192-26719cc6-7f49-4650-bcd6-ddfe629cd2f7.jpg)
![unknown](https://user-images.githubusercontent.com/32972166/193721194-b69b9db9-2a47-4caa-8f86-a4aa60dcc50e.png)

**Comment by Andre601** [comment]
I believe thisissue is caused by CMI and not PlaceholderAPI itself.

Nothing in PlaceholderAPI would add extra `%` symbols. It checks if a placeholder is a valid format, if the expansion for it is present and if said expansion returns a non-null value.
If any of the above fails does PlaceholderAPI just return the unchanged String.

Tl;dr: Ask the CMI dev for help as this is most likely not a PlaceholderAPI issue.

I'll leave this issue open for now.

**Comment by Andre601** [comment]
Given this was reported on the CMI repo in the past and that the author closed it there as completed can we assume this is fixed.

If not, reply to the issue with up-to-date info.
